src/notation/qml/MuseScore/NotationScene/PercussionPanel.qml

```diff
@@ -434,6 +435,7 @@ Item {
 
             font: ui.theme.bodyFont
             text: qsTrc("notation/percussion", "Select an unpitched percussion staff to see available sounds")
+            verticalAlignment: Text.AlignVCenter
```

Comment:
I would think this doesn't have any effect if the `height` of the label is not more than its `implicitHeight`, does it?
Also, does something like `anchors.centerIn: parent` do the right thing?